### DISCUSSION TITLE
Is burnt fees a good idea with the Bitcoin?

### DISCUSSION THREAD

**Discussion by wenceslas-sanchez** [blockscout/blockscout#9720] [General]
Hi team,

I just discovered your project and I was wondering while looking for transactions if burning BTC for each block was actually a good idea? I do understand the meaning in Ethereum but why did you implemented this here ?

**Comment by wenceslas-sanchez** [comment]
@GimluCom @sl1depengwyn Do you have a the time to look at the question above?

**Comment by GimluCom** [comment]
Blockscout is not a chain or a network. Please contact the team of the chain/network you are having issues with.
